com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/PathTracing.cs

```diff
@@ -271,7 +271,7 @@ PathTracingParameters PreparePathTracingParameters(HDCamera hdCamera)
         {
             PathTracingParameters parameters = new PathTracingParameters();
 
-            parameters.pathTracingShader = m_Asset.renderPipelineRayTracingResources.pathTracing;
+            parameters.pathTracingShader = HDRenderPipelineGlobalSettings.instance.renderPipelineRayTracingResources.pathTracing;
```

Comment:
Suggestion: This is a partial class of HDRenderPipeline so instead of using HDRenderPipelineGlobalSettings.instance
 we could use m_GlobalSettings? I think it's true for a lot of other places as many subsystems of HDRP are just partial class implementation.

Reply:
we could, i did not notice it was a partial class of HDRenderPipeline, do you want me to fix this for this PR or add it to a polish task later?

Reply:
It can probably be done later yes.